src/profile-logic/import/chrome.js

```diff
@@ -824,6 +835,16 @@ function extractMarkers(
             endTime: time + duration,
           });
         } else if (event.ph === 'B' || event.ph === 'E') {
+          if (event.ph === 'B') {
+            markers.startTime.push(time);
```

Comment:
It would be good to add some comments that mention "B" means begin and "E" means end. It took me some time to get it 😄

Reply:
Hah, a classic case of reading the spec and forgetting how arcane some things are.